distdir_deps.bzl

```diff
@@ -11,7 +11,17 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""List the distribution dependencies we need to build Bazel."""
+"""
+List the distribution dependencies we need to build Bazel.
+
+Note for Bazel users: This list is not intended to tell you what version
+you should use for each dependency. Indeed in this list you will find that
```

Comment:
I don't quite understand this. This is only for Bazel itself, not a recommendation for users, right?

Reply:
yes, the idea would be that if a user comes here to this code to try to figure out what version to use. This should kick the user away and say that this is not the place to look. The question is how to phrase it.
I already found several users pointing me to this file to justify the usage of some specific version of a dependency

Reply:
Oh, sorry. I missed the "not" in "This list is is not intended"..

Reply:
Maybe I should have put a **NOT** ;-)